test/libsolidity/semanticTests/functionTypes/selector_assignment_expression_2.sol

```diff
@@ -0,0 +1,14 @@
+contract D {
+    function f() external {}
+}
```

Comment:
Like I said in https://github.com/ethereum/solidity/pull/14366#discussion_r1266909341, I thought this case would be diffenent in a more significant way :) In that case I'd actually drop it.